Source/DafnyDriver/CSVTestLogger.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+
+namespace Microsoft.Dafny {
+  public class CSVTestLogger : ITestLoggerWithParameters {
+
+    private readonly List<TestResult> results = new();
+
+    public void Initialize(TestLoggerEvents events, string testRunDirectory) {
+    }
+
+    public void Initialize(TestLoggerEvents events, Dictionary<string, string> parameters) {
+      events.TestResult += TestResultHandler;
+      events.TestRunComplete += TestRunCompleteHandler;
+    }
+
+    private void TestResultHandler(object sender, TestResultEventArgs e) {
+      results.Add(e.Result);
+    }
+
+    private int? GetResourceCount(TestResult result) {
+      return result.GetPropertyValue<int?>(BoogieXmlConvertor.ResourceCountProperty, null);
+    }
+
+    private void TestRunCompleteHandler(object sender, TestRunCompleteEventArgs e) {
+      foreach (var result in results.OrderByDescending(GetResourceCount)) {
```

Comment:
We can just make this sort by duration instead for now. I'm amused that I made it sort by resource count without actually adding the column :)

Reply:
Ah, you weren't actually that strange. :) I removed that column, just not the code to track it.